src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -164,8 +164,36 @@ end
 
 end
 
+variables (ι 𝕜)
+
+/-- `pi_Lp.linear_equiv` upgraded to a continuous linear map between `euclidean_space 𝕜 ι`
+and `ι → 𝕜`.
+
+TODO : This should be generalized to `pi_Lp` with finite dimensional factors. -/
```

Comment:
I think these TODOs should be regular comments, as opposed to being part of the docstring.